# FTP::Close
#
# Call FTP::Open first.

string file_path
---
bool success
